### ISSUE TITLE
Latest change might be a breaking change...

### ISSUE THREAD

**Issue by benlamptey** [drupalauth/simplesamlphp-module-drupalauth#93]
**Describe the bug**
Warning: Undefined array key "drupalauth:External:UserID" in /var/lib/jenkins/workspace/website/vendor/simplesamlphp/simplesamlphp/modules/drupalauth/lib/Auth/Source/External.php on line 279

```
SimpleSAML\Error\Error: UNHANDLEDEXCEPTION

Backtrace:
1 www/_include.php:17 (SimpleSAML_exception_handler)
0 [builtin] (N/A)
Caused by: SimpleSAML\Error\Exception: User not authenticated after login page.
Backtrace:
3 modules/drupalauth/lib/Auth/Source/External.php:287 (SimpleSAML\Module\drupalauth\Auth\Source\External::resume)
2 modules/drupalauth/www/resume.php:12 (require)
1 lib/SimpleSAML/Module.php:266 (SimpleSAML\Module::process)
0 www/module.php:10 (N/A)
```
**To Reproduce**
Click SSO on the SP

**Expected behavior**
A drupal user successfully logged in.

**Screenshots**

Are we supposed to set `drupalauth:External:UserID` somewhere? if so where?

**Comment by RoSk0** [comment]
It would be easier to help if you would provide all information specified in template...

Please extend your description with the following : 
```
**Please complete the following information:**

- PHP version in use
- Drupal core version
- SimpleSAMLphp version
- webserver version and configuration
- drupalauth/simplesamlphp-module-drupalauth version
- authntication source plugin used
- drupalauth4ssp module version, if used
```

**Comment by RoSk0** [comment]
This is fixed in the version 1.9.3 released just now. Please update.
